packages/calcite-components/src/components/menu-item/menu-item.tsx

```diff
@@ -465,6 +513,7 @@ export class CalciteMenuItem implements LoadableComponent, T9nComponent, Localiz
             [CSS.container]: true,
             [CSS.isParentVertical]: this.topLevelMenuLayout === "vertical",
           }}
+          ref={this.setTransitionEl}
```

Comment:
Wouldn't this overwrite the other transitionEl set on line 472?

Reply:
I'm not exactly sure where the right place for it is. It's kind of a Russian doll situation. When you slot items into `menu-items` it renders `menu`. So you could set it on the `container` for menu-item, or `menu` itself?